java/ql/test/query-tests/security/CWE-022/semmle/tests/NewPathInjection/PathInjection/src/main/java/com/PathInjection/CommonsIOPathInjection.java

```diff
@@ -0,0 +1,38 @@
+package com.PathInjection;
+
+import java.io.*;
+import java.nio.channels.AsynchronousFileChannel;
+import java.nio.file.*;
+import java.nio.file.attribute.FileAttribute;
+import java.util.Set;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+class fileAttr implements FileAttribute<String> {
+  public String name() {
+    return "file";
+  }
+
+  public String value() {
+    return "value";
+  }
+}
+
+public class CommonsIOPathInjection {
```

Comment:
Why is there a Commons-IO test being added but not any new models for that library? Is this just a missing test, or is there a missing models file?